PlayerMovement.cs

```diff
@@ -184,87 +170,50 @@ public class PlayerMovement : MonoBehaviour {
         orientation.transform.localRotation = Quaternion.Euler(0, desiredX, 0);
     }
 
-    private void CounterMovement(float x, float y, Vector2 mag) {
-        if (!grounded || jumping) return;
+    void ApplyFriction() {
```

Comment:
Since we're storing user input and can get velocity relative to camera orientation, this method can omit params